packer/scripts/50-services.sh

```diff
@@ -7,3 +7,6 @@ systemctl start fail2ban
 
 systemctl enable caddy
 systemctl start caddy
+
+systemctl enable ufw
```

Comment:
I'm out of ideas now. Somehow DigitalOcean's own images manage to have UFW on by default, but none of these commands manage to do the same for us.

Reply:
Hmm, I dunno either. We should remove things that don't work, though.

Reply:
Yeah, I kept these lines so I can start this conversation with that note 😄  I'll create a ticket and have them look at it. How does this sound?

Reply:
That sounds good. It might be a good idea to show how you know that ufw is not enabled too, just so they can verify.

Reply:
Fixed! with d9d5273